middlewares/requests/admin/admin-class-store.request.js

```diff
@@ -0,0 +1,32 @@
+const { body, validationResult, param } = require('express-validator');
+const cohorts = require('../../../dao/cohort.dao');
+
+const adminClassStoreRequest = [
+  body('name').isLength({ min: 1, max: 255 }).trim().escape(),
+  body('subject').isLength({ min: 1, max: 255 }).trim().escape(),
+  param('cohortId').custom(async (value) => {
+    try {
+      console.log(value);
+      const cohort = await cohorts.findCohortById(value);
+      if (!cohort) {
+        // eslint-disable-next-line prefer-promise-reject-errors
+        return Promise.reject('cohort ID is invalid');

```

Comment:
Recommended change: 'Cohord ID is invalid.'

Reply:
Sorry for being picky. This should be 'Cohort ID is invalid.'

Reply:
LOL